mithril-aggregator/src/services/usage_reporter.rs

```diff
@@ -84,13 +95,22 @@ impl UsageReporter {
 
         self.last_reported_metrics = metrics;
 
-        for (name, origin_values) in delta {
-            for (origin, value) in origin_values {
+        for (name, metrics_map) in delta {
+            for (_origin, metric_map) in metrics_map {
```

Comment:
I think `origin` is not an appropriate name as this is not what is represented here